cluster/gce/manifests/konnectivity-server.yaml

```diff
@@ -51,5 +51,21 @@ spec:
       path: /etc/srv/kubernetes/pki
   - name: konnectivity-uds
     hostPath:
-      path: /etc/srv/kubernetes/konnectivity
+      path: /etc/srv/kubernetes/konnectivity-server
       type: DirectoryOrCreate
+---
```

Comment:
it's weird to have a second blob in this .yaml, when you made a separate file above?

Reply:
Each cluster addon has its own directory but all the manifests share the same directory (at least in gce). Didn't want to pollute the manifests directory.

Reply:
I see, ok.